modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="compliance_profile_types_{context}"]
+= Compliance Operator profile types
+
+Compliance Operator rules are organized into profiles. Profiles can target the {product-title} Platform or Nodes, and some standards include `rhcos4` Node profiles.
+
+Platform:: Platform profiles evaluate your {product-title} cluster components. For example, confirming APIServer configurations are using strong encryption cyphers is Platform-level rule.
+
+Node:: Node profiles for evaluating the OpenShift or {op-system} configuration of each host. There are `ocp4` Node profiles and `rhcos4` Node profiles. `ocp4` Node profiles evaluate the OpenShift configuration of each host. For example, confirming kubeconfig files have the correct permissions to meet a compliance standard is a Node-level rule. `rhcos4` Node profiles evaluate the {op-system-first} configuration of each host. For example, confirming the SSHD service is configured to disable password logins is a Node-level rule.
+
+[IMPORTANT]
+====
+For benchmarks that have Node and Platform profiles, such as PCI-DSS, you must run both profiles in your {product-title} environment.

```

Comment:
Is it better to use below sentence?
`For benchmarks that have `ocp4` Platform and `ocp4` Node profiles, such as PCI-DSS, you must run both profiles in your {product-title} environment.`

Reply:
I specifically worded it this way to make it clear to readers that regardless of whether a benchmark includes rhcos4 or ocp4 nodes, the profiles must be used with platform profiles. And I used PCI-DSS as an example because it was the first one I could think of that used both. 

I'd like to keep it this way, but we can make changes later if you think it's worth re-addressing later. 

In the meantime I'll go ahead and get this content peer reviewed and merged.